samples/client/echo_api/python/openapi_client/api_client.py

```diff
@@ -386,21 +383,32 @@ def sanitize_for_serialization(self, obj):
             for key, val in obj_dict.items()
         }
 
-    def deserialize(self, response_text, response_type):
+    def deserialize(self, response_text: str, response_type: str, content_type: str):
```

Comment:
`content_type` should be optional since you test for `None` a few lines after:

```suggestion
    def deserialize(self, response_text: str, response_type: str, content_type: Optional[str]):
```